packages/frontend-2/lib/common/composables/toast.ts

```diff
@@ -1,93 +1,60 @@
+import { useTimeoutFn } from '@vueuse/core'
 import type { Optional } from '@speckle/shared'
 import type { ToastNotification } from '@speckle/ui-components'
-import { useTimeoutFn } from '@vueuse/core'
 import { ToastNotificationType } from '@speckle/ui-components'
 import { useSynchronizedCookie } from '~/lib/common/composables/reactiveCookie'
-import { nanoid } from 'nanoid'
 
 /**
  * Persisting toast state between reqs and between CSR & SSR loads so that we can trigger
  * toasts anywhere and anytime
  */
 const useGlobalToastState = () =>
-  useSynchronizedCookie<Optional<ToastNotification[]>>('global-toast-state')
+  useSynchronizedCookie<Optional<ToastNotification>>('global-toast-state')
```

Comment:
i'm wondering what will happen when the page loads in for someone who already has this cookie set with an array of toast notifications, hopefully the code doesn't break (expects a single object, finds an array).

its probably fine, but we should probably validate this assumption locally